sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/receiptverification/_version.py

```diff
@@ -0,0 +1,7 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------
+
+VERSION = "1.1.0"
```

Comment:
Is there a reason that the version of the submodule is maintained separately?

Reply:
I was following the same example of the `azure.confidentialledger.certificate` sub-module (see ) for consistency. If that's not required or not needed, I can definitely remove it.

Reply:
Removed in https://github.com/Azure/azure-sdk-for-python/pull/27265/commits/e558b7840ceebc54673364d5886280c06a7e1ad1